### PR TITLE
Reminder option for Gbot

### PR DESCRIPTION
**Pull request by SyedSibtainRazvi**
# Related Issue

- https://github.com/code-mancers/gather-bot/issues/170

# functionalities Added or Changed

- Added reminder to schema and create standup

# Additional Info

- 

# Checklist

- [ ] Tests
- [ ] Uploaded screenshots

# Screenshots
reminder

### CODE REVIEW

File: server/prisma/schema.prisma [hunk 1]
```diff
@@ -47,4 +47,5 @@ model Standup {
   workspace    Workspace[]
   originalTime String?
   days         Int[]
+  reminder     Int[]
```
Comment (SujayPrabhu96) [inline server/prisma/schema.prisma:50]:
```suggestion
  reminder     Int
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/prisma/schema.prisma (hunk 1 above)
Comment (SyedSibtainRazvi) [reply server/prisma/schema.prisma]: Resolved and updated the service as well as test cases.

File: server/src/services/standups.service.ts [hunk 2]
```diff
@@ -131,6 +134,7 @@ export const postStandup = async ({
         },
         days: [...days],
         originalTime: time,
+        reminder: reminder,
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:137]:
```suggestion
        reminder,
```

File: server/src/services/standups.service.ts [hunk 3]
```diff
@@ -182,6 +186,7 @@ export const getStandup = async (id: string) => {
     title: standup?.title,
     originalTime: standup?.originalTime,
     days: standup?.days,
+    reminder: standup?.reminder,
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:189]: remove optional chaining

File: server/src/services/standups.service.ts [hunk 4]
```diff
@@ -221,6 +227,7 @@ export const updateStandup = async (
       data: {
         title: title,
         days: [...days],
+        reminder: reminder,
```
Comment (SujayPrabhu96) [inline server/src/services/standups.service.ts:230]:
```suggestion
        reminder,
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/services/standups.service.ts (hunk 4 above)
Comment (SyedSibtainRazvi) [reply server/src/services/standups.service.ts:230]: Done

File: server/src/services/standups.service.ts (hunk 3 above)
Comment (SyedSibtainRazvi) [reply server/src/services/standups.service.ts:189]: Can be removed once we handled it in FE, else it says our standup is possibly null

File: server/src/services/standups.service.ts (hunk 2 above)
Comment (SyedSibtainRazvi) [reply server/src/services/standups.service.ts:137]: done

**Review by SujayPrabhu96 — APPROVED** [review]

File: server/src/services/standups.service.ts (hunk 3 above)
Comment (SujayPrabhu96) [reply server/src/services/standups.service.ts]: Update: This has already been fixed in PR: 217

**Pull request merged**